corehq/apps/app_manager/xform.py

```diff
@@ -729,8 +729,7 @@ def odk_intents(self):
     def text_references(self, lang=None):
         # Accepts lang param for consistency with image_references, etc.,
         # but current text references aren't language-specific
-        nodes = self.findall('{h}head/{odk}intent[@class="org.commcare.dalvik.action.PRINT"]'
-                             '/{f}extra[@key="cc:print_template_reference"]')
+        nodes = self.findall('{h}head/{odk}intent[@class="org.commcare.dalvik.action.PRINT"]')
```

Comment:
I think you can instead delete this function, and the calling code [here](https://github.com/dimagi/commcare-hq/blob/bbb06e7b0c5ed94064e8e6094a7467fe3af4e656/corehq/apps/hqmedia/models.py#L746-L748), which looks like it's for including print templates as part of the app's multimedia.

Reply:
Thanks for the suggestion here @orangejenny 

Before going ahead with the removal I am trying to get some confidence.

I don't have context around this code here, so I am looking at history of updates to this function.

I see an [update](https://github.com/dimagi/commcare-hq/pull/7764/files) to it, way before case detail print feature was added, so I am assuming that this is probably used elsewhere too?
I am thinking of the [form print feature](https://github.com/dimagi/commcare-hq/pull/7594) which was added roughly at the same time as the update I linked above, which could be using this function here.

Reply:
oh wow, well, I assumed "print_template_reference" is for "print_template".
But there is enough proof that I should put it back.